packages/building-webpack/modern-config.js

```diff
@@ -67,6 +92,10 @@ module.exports = userOptions => {
             },
           },
         },
+        {
+          test: /\.html$/,
```

Comment:
Make sure to only match the actual `index.html` rather than any `.html` file.

Reply:
only add the loader if needed and only match the input file 💪